testapi.pm

```diff
@@ -197,23 +192,21 @@ influenced.
 
 =cut
 
-sub record_soft_failure {
-    my ($reason) = @_;
+sub record_soft_failure ($reason) {
     bmwqemu::log_call(reason => $reason);
 
     $autotest::current_test->record_soft_failure_result($reason);
 }
 
-sub _is_valid_result {
-    my ($result) = @_;
+sub _is_valid_result ($result) {
     return $result =~ /^(ok|fail|softfail)$/;
 }
```

Comment:
```suggestion
sub _is_valid_result ($result) { $result =~ /^(ok|fail|softfail)$/ }
```

Reply:
I didn't want to change something unrelated to signatures

Reply:
ok, changed